content/guides/api-calls/api-versioning-strategy.md

```diff
@@ -69,10 +71,20 @@ We strongly recommend updating your apps to make requests to the latest stable A
 
 If your request doesn't include a version, the API defaults to the initial Box API version—the version available before 
 year-based versioning was introduced. However, relying on this behavior is not recommended when adopting deprecated 
-changes. To ensure consistency, always specify the API version with each request. By making your application 
+changes. To ensure consistency, always specify the API version, with each request. By making your application 
 version-aware, you anchor it to a specific set of features, ensuring consistent behavior throughout the supported 
 timeframe.
 
+### Endpoint marking
+
+To keep you informed about the current API state, and improve the readability of the versioned API reference, the affected endpoints are marked with the following terms: 
+
+* **Beta**: Endpoints marked with **beta**, are offered subject to Box’s Main Beta Agreement, meaning the available capabilities may change at any time. Although beta endpoints not the same as versioned endpoints, they are a part of API lifecycle.
+
+* **Latest version**: The latest version applies to APIs that are already versioned. **Latest version** marks the most recent API version of an endpoint.
```

Comment:
What about adding 'stable' here? `...marks the most recent stable API version...`